src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsEngine.java

```diff
@@ -179,8 +179,11 @@ private VariantContext regenotypeVC(final VariantContext originalVC, final Refer
         // We could theoretically make 2 passes to re-create the genotypes, but that gets extremely expensive with large sample sizes.
         if (result.isPolymorphicInSamples()) {
             // For polymorphic sites we need to make sure e.g. the SB tag is sent to the annotation engine and then removed later.
-            VariantContext preannotated = new VariantContextBuilder(result).genotypes(cleanupGenotypeAnnotations(result, false)).make();
-            return annotationEngine.annotateContext(preannotated, features, ref, null, a -> true);
+            final VariantContextBuilder vcBuilder = new VariantContextBuilder(result);
+            //don't count sites with no depth and no confidence towards things like AN and InbreedingCoeff
+            vcBuilder.genotypes(assignNoCalls(result.getGenotypes()));
```

Comment:
I would rename ```assignNoCalls``` to something like ```assignNoCallsAnnotationExcludedGenotypes``` since otherwise it sounds like all genotypes will suffer the same fate.